### PR TITLE
[BBC-8438] ItemBigData 

### PR DESCRIPTION
**Pull request by gabrielagabriel**
## Description

Back with `ItemBigData`

## What has been done

Restore this PR: check it out

https://github.com/blablacar/ui-library/pull/750

## Things to consider

 The _overide_ issue has already been fixed on BBC App

## How it was tested

- BBC App

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 - [...]
+- **[BREAK CHANGE]** `Item`: `leftTitle` attribute can receive `string` or `node`
```
Comment (Tip-Sy) [inline CHANGELOG.md:4]: nit: usually we say `BREAKING CHANGE` 🤷‍♂️

File: src/_utils/item/Item.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export interface ItemProps extends A11yProps {
   readonly href?: string | JSX.Element
   readonly highlighted?: boolean
   readonly isClickable?: boolean
-  readonly leftTitle?: string
+  readonly leftTitle?: React.ReactNode
```
Comment (Tip-Sy) [inline src/_utils/item/Item.tsx:22]: I guess it should be: `string | React.ReactNode` here

Comment (Tip-Sy) [reply src/_utils/item/Item.tsx:22]: - [ ] Update type?

File: src/_utils/item/Item.tsx [hunk 3]
```diff
@@ -101,6 +101,24 @@ export const Item = (props: ItemProps) => {
 
   const getTextColor = (textColor: string) => (disabled ? color.gray : textColor)
 
+  const getLeftTitle = (value: React.ReactNode) => {
```
Comment (Tip-Sy) [inline src/_utils/item/Item.tsx:104]: Missing return type

Comment (Tip-Sy) [reply src/_utils/item/Item.tsx:104]: Apparently we don't always add it 🤔 

- [ ] Maybe add return type then? 🤷‍♂️

**Comment by Tip-Sy** [comment]
Is it possible to have a screenshot in the description of the PR? 🙏 :)

File: src/_utils/item/Item.tsx (hunk 2 above)
Comment (gabrielagabriel) [reply src/_utils/item/Item.tsx:22]: No need. React.RectNode has intrinsic `string, number, node`...

Comment (gabrielagabriel) [reply src/_utils/item/Item.tsx:22]: ReactNode has string, number, boolean intrisically

**Comment by gabrielagabriel** [comment]
test this please

**Pull request closed without merging**